app/[locale]/faculty-and-staff/page.tsx

```diff
@@ -1,9 +1,157 @@
-import WorkInProgress from '~/components/work-in-progress';
+'use client';
+
+import { useState } from 'react';
+import { AiOutlineSearch } from 'react-icons/ai';
+
+import FacultyCard from './faculty-card';
+
+interface FacultyProps {
+  profileImage: string;
+  name: string;
+  designation: string;
+  email: string;
+  phone: string;
+  id: number;
+}
 
 export default function FacultyAndStaff({
   params: { locale },
 }: {
   params: { locale: string };
 }) {
-  return <WorkInProgress locale={locale} />;
+  const [facultyList, setFacultyList] = useState<FacultyProps[]>([
+    {
+      profileImage:
+        'https://s3-alpha-sig.figma.com/img/6698/6d20/c7aa47a52ea27a318d8cb53424a5e808?Expires=1711324800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=WNeY~2a8t85zJ-C5kvhI5xs6pVrmFYQg~9z367Cqzlpy9cn6fLBEp9IoEpsvC4wclpaWw9BZ0azdAa~t2HE0-B9gkfqGS4ABg69JE9whHNqFslgrk6yCDMG21zt9YcQShVd7OwYDLYBDAy7ChO2MWyhBfowI93P4sLxKhN~B~m44wrmCdxPbgnM60qHx8O02tfWa6L79w~FRc4FF-AzoypfBMaVveEPLFEhqfKrOjF~K3G8By3yx0-0WP1VNoHgTX06Jqw1Uf7LdB76OzGblx~2Dyn4UB9zW1bQxpBiKb8Fefk5fSStS3p2wZuKpk4dfUKkP0locrEfrt7rTEKhFFg__',
+      name: 'Arun Goel',
+      designation: 'Professor (Head of the Department)',
```

Comment:
Head of department will be a boolean which is then added next to the designation, it won't be a string that's included in the designation.